cloud_blobstore/s3.py

```diff
@@ -404,6 +405,55 @@ def get_size(
                 raise BlobNotFoundError(f"Could not find s3://{bucket}/{key}") from ex
             raise BlobStoreUnknownError(ex)
 
+    def multipart_parallel_upload(
+            self,
+            bucket: str,
+            key: str,
+            src_file_handle: typing.BinaryIO,
+            size: int,
+            part_size: int,
+            content_type: str=None,
+            metadata: dict=None,
+            parallelization_factor=8) -> typing.Sequence[dict]:
+        """
+        Upload a file object in parallel.
+        :param bucket:
+        """
+        kwargs: dict = dict()
```

Comment:
Is this type annotation actually useful?  I would assume that `kwargs = dict()` conveys the same information.

Possibly this may be more useful?
```suggestion
        kwargs: Dict[str, Any] = dict()
```